src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
> What if the API introduces a new method, but the generator doesn't change? We should be able to tell that the user is using a client library with the new method, even it's the same generator as previously.

Good point. It seems like we need both versions of generator and client. We also need to distinguish the client which uses the same client version but different versions of generators.